### ISSUE TITLE
[deployer] 2.5 deployer build leaves jars in folder after aio jar build

### ISSUE THREAD

**Issue by russdanner** [craftercms/craftercms#1347]
The only jar that should be in the bundle is the AIO (All in One Jar)
`-rw-rw-r--@  1 rdanner  staff  24297773 Feb 17  2017 crafter-studio-publishing-receiver-2.5.5-aio.jar`

The following jars are left after the build but should be removed.
````
-rw-rw-r--@  1 rdanner  staff    125146 Dec 19  2016 JavaEWAH-0.7.9.jar
-rw-rw-r--@  1 rdanner  staff      4467 Dec 19  2016 aopalliance-1.0.jar
-rw-rw-r--@  1 rdanner  staff    608376 Dec 19  2016 c3p0-0.9.1.1.jar
-rw-rw-r--@  1 rdanner  staff    233859 Dec 19  2016 commons-beanutils-1.9.2.jar
-rw-rw-r--@  1 rdanner  staff     30085 Dec 19  2016 commons-codec-1.2.jar
-rw-rw-r--@  1 rdanner  staff    588337 Dec 19  2016 commons-collections-3.2.2.jar
-rw-rw-r--@  1 rdanner  staff    751238 Dec 19  2016 commons-collections4-4.1.jar
-rw-rw-r--@  1 rdanner  staff     24283 Dec 19  2016 commons-daemon-1.0.15.jar
-rw-rw-r--@  1 rdanner  staff     59590 Dec 19  2016 commons-fileupload-1.2.2.jar
-rw-rw-r--@  1 rdanner  staff    305001 Dec 19  2016 commons-httpclient-3.1.jar
-rw-rw-r--@  1 rdanner  staff    182650 Dec 19  2016 commons-io-2.3.jar
-rw-rw-r--@  1 rdanner  staff    284220 Dec 19  2016 commons-lang-2.6.jar
-rw-rw-r--@  1 rdanner  staff    434678 Dec 19  2016 commons-lang3-3.4.jar
-rw-rw-r--@  1 rdanner  staff     38015 Dec 19  2016 commons-logging-1.0.4.jar
-rw-rw-r--@  1 rdanner  staff    101652 Feb 17  2017 crafter-commons-utilities-2.5.5.jar
-rw-rw-r--@  1 rdanner  staff     11150 Feb 17  2017 crafter-search-api-2.5.5.jar
-rw-rw-r--@  1 rdanner  staff     24976 Feb 17  2017 crafter-search-batch-indexer-2.5.5.jar
-rw-rw-r--@  1 rdanner  staff     10482 Feb 17  2017 crafter-search-client-2.5.5.jar
-rw-rw-r--@  1 rdanner  staff    313898 Dec 19  2016 dom4j-1.6.1.jar
-rw-rw-r--@  1 rdanner  staff    589512 Dec 19  2016 httpclient-4.3.3.jar
-rw-rw-r--@  1 rdanner  staff    282269 Dec 19  2016 httpcore-4.3.2.jar
-rw-rw-r--@  1 rdanner  staff   2450757 Dec 19  2016 icu4j-2.6.1.jar
-rw-rw-r--@  1 rdanner  staff    228552 Dec 19  2016 jackson-core-asl-1.9.7.jar
-rw-rw-r--@  1 rdanner  staff    780301 Dec 19  2016 jackson-mapper-asl-1.9.7.jar
-rw-rw-r--@  1 rdanner  staff     54434 Dec 19  2016 javax.activation-1.1.0.v201105071233.jar
-rw-rw-r--@  1 rdanner  staff    490811 Dec 19  2016 javax.mail.glassfish-1.4.1.v201005082020.jar
-rw-rw-r--@  1 rdanner  staff    200387 Dec 19  2016 javax.servlet-3.0.0.v201112011016.jar
-rw-rw-r--@  1 rdanner  staff     29098 Dec 19  2016 javax.transaction-1.1.1.v201105210645.jar
-rw-rw-r--@  1 rdanner  staff    226915 Dec 19  2016 jaxen-1.1.1.jar
-rw-rw-r--@  1 rdanner  staff     16519 Dec 19  2016 jcl-over-slf4j-1.7.7.jar
-rw-rw-r--@  1 rdanner  staff    153253 Dec 19  2016 jdom-1.0.jar
-rw-rw-r--@  1 rdanner  staff     21201 Dec 19  2016 jetty-continuation-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff     60281 Dec 19  2016 jetty-deploy-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff     93872 Dec 19  2016 jetty-http-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff    101449 Dec 19  2016 jetty-io-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff     39632 Dec 19  2016 jetty-jndi-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff     84011 Dec 19  2016 jetty-plus-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff     86284 Dec 19  2016 jetty-security-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff    330993 Dec 19  2016 jetty-server-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff     95535 Dec 19  2016 jetty-servlet-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff      8130 Dec 19  2016 jetty-spring-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff    268228 Dec 19  2016 jetty-util-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff    106956 Dec 19  2016 jetty-webapp-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff     37924 Dec 19  2016 jetty-xml-8.1.2.v20120308.jar
-rw-rw-r--@  1 rdanner  staff    280205 Dec 19  2016 jsch-0.1.53.jar
-rw-rw-r--@  1 rdanner  staff    481535 Dec 19  2016 log4j-1.2.16.jar
-rw-rw-r--@  1 rdanner  staff      4399 Dec 19  2016 org.eclipse.jdt.annotation-1.1.0.jar
-rw-rw-r--@  1 rdanner  staff   2162975 Dec 19  2016 org.eclipse.jgit-4.1.1.201511131810-r.jar
-rw-rw-r--@  1 rdanner  staff    660316 Dec 19  2016 quartz-2.2.1.jar
-rw-rw-r--@  1 rdanner  staff    208025 Dec 19  2016 rome-0.9.jar
-rw-rw-r--@  1 rdanner  staff     25496 Dec 19  2016 slf4j-api-1.6.1.jar
-rw-rw-r--@  1 rdanner  staff      9753 Dec 19  2016 slf4j-log4j12-1.6.1.jar
-rw-rw-r--@  1 rdanner  staff    269579 Dec 19  2016 snakeyaml-1.16.jar
-rw-rw-r--@  1 rdanner  staff    352700 Dec 19  2016 spring-aop-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    669978 Dec 19  2016 spring-beans-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    974938 Dec 19  2016 spring-context-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    135373 Dec 19  2016 spring-context-support-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    973502 Dec 19  2016 spring-core-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    204745 Dec 19  2016 spring-expression-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff     80268 Dec 19  2016 spring-oxm-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    248201 Dec 19  2016 spring-tx-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    667960 Dec 19  2016 spring-web-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff    662385 Dec 19  2016 spring-webmvc-4.0.3.RELEASE.jar
-rw-rw-r--@  1 rdanner  staff   3149655 Dec 19  2016 xalan-2.6.0.jar
-rw-rw-r--@  1 rdanner  staff   1010675 Dec 19  2016 xercesImpl-2.6.2.jar
-rw-rw-r--@  1 rdanner  staff    109318 Dec 19  2016 xml-apis-1.0.b2.jar
-rw-rw-r--@  1 rdanner  staff    124724 Dec 19  2016 xmlParserAPIs-2.6.2.jar
-rw-rw-r--@  1 rdanner  staff    108874 Dec 19  2016 xom-1.0.jar
```

**Comment by avasquez614** [comment]
Fixed in craftercms/legacy-deployer#52

**Comment by russdanner** [comment]
verified
